.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build Rucio/Butler ingestd container
+on:
+  push:
+    tags:
+    - v*
+  pull_request:
+
+env:
+  HERMESK_NAME: rucio-daemons-hermesk
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Build ingest image
+      working-directory: docker
+      run: |
+        docker-compose -f "docker-compose.yml" --env-file versions.env build $HERMESK_NAME
```

Comment:
It looks like there isn't that much in the `docker-compose.yml`.  You might be able to use https://github.com/lsst-sqre/build-and-push-to-ghcr if you go directly to the `Dockerfile`.

Reply:
I tried, but was unable to get it to work.  Plus the current version of build-and-push-to-ghcr doesn't support env (at least according to a message today on slack).  In an effort to get this out, I'm fixing the `docker-compose` to `docker compose` and staying with this for now.  I'll add that change to another ticket.